### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,11 +44,16 @@ def read(*rnames):
     name='zope.app.wsgi',
     version='4.3.1.dev0',
     url='https://github.com/zopefoundation/zope.app.wsgi',
+    project_urls={
+        'Issue Tracker': ('https://github.com/zopefoundation/'
+                          'zope.app.wsgi/issues'),
```
Comment (mgedmin) [inline setup.py:49]: +:100:

File: src/zope/app/wsgi/testlayer.py [hunk 2]
```diff
@@ -184,7 +187,8 @@ def getBody(self):
 
     def getOutput(self):
         status = self.response.status
-        status = status.encode('latin1') if not isinstance(status, bytes) else status
+        status = status.encode('latin1') if not isinstance(
+            status, bytes) else status
```
Comment (mgedmin) [inline src/zope/app/wsgi/testlayer.py:191]: Very minor, but I believe this is a one-liner for one reason only: to trick branchless coverage tracking into marking the statement as covered on both Python 2 and Python 3.

Given that we're now enabling branch coverage tracking everywhere we can stop being clever and write
```suggestion
        if not isinstance(status, bytes):
            status = status.encode('latin1')
```

File: tox.ini [hunk 3]
```diff
@@ -1,39 +1,69 @@
...
-    coverage combine
-    coverage report
+    mkdir -p {toxinidir}/parts/htmlcov
+    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
     coverage html
-    coverage xml
-# parallel mode: make sure all builds complete before we run this one
-depends = py27,py35,py36,py37,py38,pypy,pypy3
-parallel_show_output = true
+    coverage report -m --fail-under=85
```
Comment (mgedmin) [inline tox.ini:50]: Huh, only 85%?  Rewrite-if-statement-to-oneliner tricks were usually done to get to 100%, so maybe I was mistaken.

**Review by mgedmin — APPROVED** [review]
LGTM

File: tox.ini (hunk 3 above)
Comment (icemac) [reply tox.ini:50]: According to https://coveralls.io/github/zopefoundation/zope.app.wsgi `master` has also just 84 % coverage.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**